Disable xfce4 screensaver rather than light-locker (add user docs) (#2128)
alan-turing-institute/data-safe-haven: docs/source/roles/researcher/troubleshooting.md

Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
:::

> ## {{ zzz }} Idle screen lock

Reviewer: ```suggestion
## {{zzz}} Idle screen lock
```
I'm not sure the code that replaces this with `:zzz:` and then replaces that with an emoji will work correctly with spaces.

Author: The emoji did render locally, let's change it to avoid whitespace problems and stay consistent though :+1:.

By default, the Xfce desktop will enter a screensaver and lock the screen after idling for five minutes.
This will require entering your password to unlock and is in addition to authenticating with Guacamole and the lock screen on your own computer.

You can disable the Xfce lock screen if you find it is unnecessary and slows your work.

> First, open Screensaver Preferences either by nagivating to **{guilabel}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.

Reviewer: ```suggestion
First, open Screensaver Preferences either by navigating to **{guilabel}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.
```

> First, open Screensaver Preferences either by nagivating to **{guilabel}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.

Reviewer: Should this whole section be: (a) a list and (b) wrapped in a dropdown?

Author: Yes, I think both makes it more consistent and looks better.

> First, open Screensaver Preferences either by nagivating to **{guilabel}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.

Reviewer: Is this actually a `menuselection`?

Author: Yes :tada:

On the **{guilabel}`Screensaver`** tab you can disable the screensaver using the **{guilabel}`Enable Screensaver`** toggle.

> Alternatively, you can navigate to the **{guilabel}`Lock Screen`** tab and disable the lock screen or locking when the screensaver runs with the **{guilabel}`Enable Lock Screen`** or **{guilabel}`Lock Screen with Screensaver`** toggles respectively.

Reviewer: For things that are GUI elements but not actually buttons, I think we've just been using bold text.

Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
:::

## {{zzz}} Idle screen lock

By default, the Xfce desktop will enter a screensaver and lock the screen after idling for five minutes.
This will require entering your password to unlock and is in addition to authenticating with Guacamole and the lock screen on your own computer.

You can disable the Xfce lock screen if you find it is unnecessary and slows your work.

:::{admonition} Disabling the lock screen
:class: dropdown note

> - First, open Screensaver Preferences either by navigating to **{menuselection}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.

Reviewer: ```suggestion
- First, open Screensaver Preferences by either:
    - navigating to **{menuselection}`Applications --> Settings --> Screensaver Preferences`**
    - or running `xfce4-screensaver-preferences` on the command line.
```

- On the **Screensaver** tab you can disable the screensaver using the **Enable Screensaver** toggle.

> - Alternatively, you can navigate to the **Lock Screen** tab and disable the lock screen or locking when the screensaver runs with the **Enable Lock Screen** or **Lock Screen with Screensave`** toggles respectively.

Reviewer: ```suggestion
- Alternatively, you can navigate to the **Lock Screen** tab to:
    - disable the lock screen with the **Enable Lock Screen** toggle
    - and/or disable locking when the screensaver runs with the **Lock Screen with Screensaver** toggle
```
